src/main/java/com/rettichlp/unicacityaddon/events/faction/fbi/DrugVaultMessageEventHandler.java

```diff
@@ -0,0 +1,77 @@
+package com.rettichlp.unicacityaddon.events.faction.fbi;
+
+import com.rettichlp.unicacityaddon.base.config.ConfigElements;
+import com.rettichlp.unicacityaddon.base.registry.annotation.UCEvent;
+import com.rettichlp.unicacityaddon.base.text.ColorCode;
+import com.rettichlp.unicacityaddon.base.text.Message;
+import com.rettichlp.unicacityaddon.base.text.PatternHandler;
+import net.minecraftforge.client.event.ClientChatReceivedEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+import java.util.regex.Matcher;
+
+/**
+ * @author Gelegenheitscode
+ */
+
+@UCEvent
+public class DrugVaultMessageEventHandler {
+    @SubscribeEvent
+    public void onClientChatReceived(ClientChatReceivedEvent e) {
+        if (!ConfigElements.getDrugVaultMessageActivated())  {
+            return;
+        }
+        String msg = e.getMessage().getUnformattedText();
+
+        Matcher m = PatternHandler.DRUG_VAULT_DROP.matcher(msg);
```

Comment:
Fast alle Matcher werden ähnlich wie das Pattern benannt: Müsste in dem Fall `drugVaultDropMatcher` heißen